### PR TITLE
insomnium 0.2.3-a (new cask)

### PR DESCRIPTION
**Pull request by pavelzw**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/i/insomnium.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  sha256 "3a561fa0d3413d23180d5488b0034a5bbcb2d433e4600e31d002e0ca2a52d9df"
+
+  url "https://github.com/ArchGPT/insomnium/releases/download/core%40#{version}/Insomnium.Core-#{version}.signed.dmg"
+  name "Insomnium"
+  desc "HTTP & GraphQL Client. FOSS Fork of Insomnia. 100%-Local, no user login, etc"
+  homepage "https://github.com/ArchGPT/insomnium"
+
+  livecheck do
+    url :url
+  end
```
Comment (bevanjkay) [inline Casks/i/insomnium.rb:12]: The `livecheck` block can be omitted here as the default `git` strategy will be inferred from the `url`.

File: Casks/i/insomnium.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+cask "insomnium" do
+  version "0.2.3-a"
+  sha256 "3a561fa0d3413d23180d5488b0034a5bbcb2d433e4600e31d002e0ca2a52d9df"
+
+  url "https://github.com/ArchGPT/insomnium/releases/download/core%40#{version}/Insomnium.Core-#{version}.signed.dmg"
+  name "Insomnium"
+  desc "HTTP & GraphQL Client. FOSS Fork of Insomnia. 100%-Local, no user login, etc"
```
Comment (bevanjkay) [inline Casks/i/insomnium.rb:7]:
```suggestion
  desc "HTTP and GraphQL Client"
```
The above is in line with the [`desc` guidelines](https://docs.brew.sh/Cask-Cookbook#stanza-desc).

**Review by bevanjkay — CHANGES_REQUESTED** [review]
Thank you for contributing to `homebrew-cask` @tomtt - please see my review comments below.

**Review by bevanjkay — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @pavelzw!

**Pull request merged**